code/modules/mob/living/carbon/human/human.dm

```diff
@@ -112,6 +115,9 @@
 
 	if(new_species_name)
 		s_tone = random_skin_tone(new_species_name)
+	skin_r = rand(0,255)
```

Comment:
`skin_r`, `skin_g`, and `skin_b` are not used unless the human's species datum has the `MULTICOLOR` anatomical flag.